sequencer/src/api.rs

```diff
@@ -1356,7 +1356,7 @@ mod api_tests {
         let avidm_param = init_avidm_param(2).unwrap();
         let weights = vec![1u32; 2];
 
-        let ns_table = parse_ns_table(payload.byte_len().as_usize(), &payload.encode());
+        let ns_table = parse_ns_table(payload.byte_len().as_usize(), &payload.ns_table().encode());
```

Comment:
I though, we always pass metadata bytes here, but I am not sure about this change

Reply:
So `ns_table()` and metadata are the same thing.  Both have type `NsTable`.  What are you unsure about here?